.circleci/build.sh

```diff
@@ -156,9 +172,10 @@ echo "  > Installing Drupal into SQLite database ${DB_FILE}."
 "${BUILD_DIR}/vendor/bin/drush" -r "$(pwd)/${BUILD_DIR}/web" status
 
 echo "  > Symlinking module code."
-rm -rf "${BUILD_DIR}/web/modules/${MODULE}"/* > /dev/null
```

Comment:
why was this changed?

Reply:
Hi @AlexSkrypnyk 
Because phpcs, phpstan or phpunit config in Drupal project usually use [web-root]/modules/custom. My idea is want to keep the changes of those files as minimal.
https://github.com/drevops/drevops/blob/main/phpunit.xml
https://github.com/drevops/drevops/blob/main/phpstan.neon
https://github.com/drevops/drevops/blob/main/phpcs.xml